project/Dependencies.scala

```diff
@@ -687,6 +686,10 @@ object Dependencies {
     "org.scala-lang.modules" %% "scala-collection-compat" % scalaCollectionCompatV,
   )
 
+  private val asyncHttpClientOverrides = List(
+    "org.asynchttpclient" % "async-http-client" % "2.10.5",
```

Comment:
Since this is a transitive dependency, at some point the libraries we upgrade to should use 2.10.5 or newer. How can we know when to remove this workaround?

Reply:
Yeah good question. Judging by the fact there seem to have been no new versions of sttp published in over 2 years, I'm not sure if/when this is going to ever happen. 🙁 

Also unfortunately the latest sttp patch versions all use versions of async-http-client older than the requested minimum of 2.10.4, so upgrading the minor version would not currently help here:

* 1.5.19 [uses 2.10.0](https://mvnrepository.com/artifact/com.softwaremill.sttp/async-http-client-backend_2.12/1.5.19)
* 1.6.8 [uses 2.10.2](https://mvnrepository.com/artifact/com.softwaremill.sttp/async-http-client-backend_2.12/1.6.8)
* 1.7.2 [uses 2.10.3](https://mvnrepository.com/artifact/com.softwaremill.sttp/async-http-client-backend_2.12/1.7.2)

Reply:
What's the delta between async-http-client 2.10.0 and 2.10.5? Is it worth moving to a newer version of sttp to make that delta as small as possible?

Reply:
We chatted in person. Summary: sttp 1.6+ introduces API changes that would require non-trivial code changes. Also, updating cats (which we want to do soon) will push on this anyway, so it is not worth moving to a newer version of sttp at this time.